docs/content/get-started/installation/aperture-cli/aperture-cli.md

```diff
@@ -36,12 +36,19 @@ chmod +x "\${BIN}/aperturectl"
 );
 ```
 
-The Aperture CLI is available as an executable program for all major platforms,
-the binaries can be downloaded from GitHub <a
+The Aperture CLI provides a convenient way to interact with Aperture. It can be
+used to generate a policy, apply a policy, list policies, and perform other
+operations such as listing connected agents, building binaries for agents and
```

Comment:
instead of referring to bulid command, we can refer to flowcontrol control-points or some other commands which can be used by the users.

Also there is an additional `The` at the end of this para

Reply:
It's just a brief description but not a holistic view of what aperture can do. I can remove the build part and change it to  flow control thing.

will remove the extra `the`